packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -116,3 +137,184 @@ startTelemetry(telemetryConfig);
+
+> Note: All telemetry being visualized below is generated from clients without any intervention from the server. The accuracy might be impacted due to inherent nature of client telemetry and data being lost due to faulty clients or lack of connectivity before the telemetry is fully pushed out. We recommend not using this telemetry for business metrics, but rather use it for operational metrics and diagnosis of issues.
+
+### Accessing App Insights Portal
+
+Before we can query, we must first navigate to your Azure App Insights telemetry page. To do this, go to your Azure App Insights Instance and click on the “Logs” tab under Monitoring.
+
+![Logs on App Insights Portal](readme_images/telemetry_1.png)
+
+Now, close out the “Queries” pane if it showed up for you and you will be in the view where we can execute our queries. Note that if you are using the Fluid Azure App Insights logger, your telemetry data will be available in the “customEvents” table. 
+
+### Queries and results
+
+1. Session information
+
+The following query provides a table of data that can give you a quick overview of information about sessions for your application. It includes the Id of the container being interacted with, the number of collaborators and the length of each session. Note that query provides session id’s but these values do not actually exist in the telemetry, it is a concept we have derived from the data; see the intro paragraph for more information on sessions.
+
+```sql
+let sessionGap = 5m; 
+let sessionGapSeconds = toint(sessionGap / 1s); 
+customEvents 
+| extend containerId = tostring(customDimensions.containerId), containerInstanceId = tostring(customDimensions.containerInstanceId ) 
+| where name startswith "fluidframework.container"
+| extend containerIdTimestamp = strcat(containerId, timestamp) 
+| sort by containerIdTimestamp asc 
+| extend prevTimestamp = prev(timestamp), prevContainerId = prev(containerId) 
+| extend Diff = datetime_diff("second", timestamp, prevTimestamp) 
+| extend IsNewPeriod = iif(prevContainerId != containerId or Diff > (sessionGapSeconds) or isnull(Diff), 1, 0) 
+| extend SessionId = row_cumsum(IsNewPeriod) 
+| summarize NumCollaborators = dcount(containerInstanceId), StartTime = min(timestamp), EndTime = max(timestamp) by SessionId, containerId 
+| extend PeriodDurationInMinutes = datetime_diff("minute", EndTime, StartTime) 
+| project SessionId, containerId , NumCollaborators, PeriodDurationInMinutes, StartTime, EndTime 
```

Comment:
```suggestion
| summarize NumContainers = dcount(containerInstanceId), StartTime = min(timestamp), EndTime = max(timestamp) by SessionId, containerId 
| extend PeriodDurationInMinutes = datetime_diff("minute", EndTime, StartTime) 
| project SessionId, containerId , NumContainers, PeriodDurationInMinutes, StartTime, EndTime 
```

We might want to rename this from Collaborators to NumContainers, or just make it clear that its not like unique people. (FYI if you make this update you'll need an updated image. I can provide if needed)

Reply:
![image](https://github.com/microsoft/FluidFramework/assets/105244057/1c5dfce5-e8e7-46e0-87e5-1b5731385aa6)